apis/Google.Bigquery.V2/Google.Bigquery.V2/BigqueryClientImpl.Queries.cs

```diff
@@ -69,19 +69,38 @@ public override BigqueryJob CreateQueryJob(string sql, CreateQueryJobOptions opt
             return new BigqueryJob(this, job);
         }
 
-        /// <inheritdoc />
-        public override BigqueryResult GetQueryResults(JobReference jobReference, GetQueryResultsOptions options = null)
+        /// <inheritdoc />        
+        public override BigqueryQueryJob PollQueryUntilCompleted(JobReference jobReference, GetQueryResultsOptions options = null, PollJobOptions pollOptions = null)
         {
             GaxRestPreconditions.CheckNotNull(jobReference, nameof(jobReference));
+            pollOptions?.Validate();
+
+            DateTimeOffset? deadline = pollOptions?.GetEffectiveDeadline() ?? DateTimeOffset.MaxValue;
```

Comment:
Why is this a DateTimeOffset, rather than a DateTime?
Can't we simply insist all datetime's are UTC?